requirements.txt

```diff
@@ -13,3 +13,4 @@ uvicorn[standard]==0.20.0
 redis==4.3.5
 icalendar == 5.0.4
 asyncpg==0.27.0
+numpy==1.24.2
```

Comment:
It would be nice if we could remove numpy as a dependency, even if the added weight is negligible. Maybe adding a comment explaining where numpy is used could help

Reply:
Numpy was used for the random.choice method when drawing winning tickets. It's now replaced by random.sample (random is part of the standard library)